demos/artprize/app.module.js

```diff
@@ -0,0 +1,72 @@
+/*globals localStorage, angular*/
+
+// App Module
+// ----------
+var app = angular.module("blockforge",[
+  "ngRoute"
+]);
+
+// Constants
+// ---------
+app.constant('constants', {
+  // Titles
+  // ------
+  DASHBOARD_TITLE: 'Blockforge:Dashboard',
+  HOME_TITLE     : 'Blockforge',
+
+  // Paths and Routes
+  // ----------------
+  DASHBOARD_PATH: '/dashboard',
+  HOME_PATH     : '/',
+
+  // Request Types
+  // -------------
+  GET           : 'GET',
+  POST          : 'POST',
+  DELETE        : 'DELETE',
+  PATCH         : 'PATCH',
+  PUT           : 'PUT'
+});
+
+// Routes
+// ------
+app.config(function($routeProvider) {
+
+  // Route Provider
+  // --------------
+  $routeProvider
+
+  // Root
+  // ----
+  .when('/artprize/', {
+    templateUrl: 'controllers/root/root.template.html',
+    controller : 'blockforgeRoot'
+  })
```

Comment:
This isn't quite working yet, but we will get it working soon.  It may require moving this demo to the root or something.  Angular is being weird.